assets/js/payment-methods-demo/stripe-utils/load-stripe.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { loadStripe } from '@stripe/stripe-js';
+
+/**
+ * Internal dependencies
+ */
+import { getApiKey } from './utils';
+
+const stripePromise = new Promise( ( resolve ) => {
+	let stripe = null;
+	try {
+		stripe = loadStripe( getApiKey() );
+	} catch ( error ) {
+		// eslint-disable-next-line no-console
```

Comment:
Will be be a customer facing error or a silent one?

Reply:
If stripe fails to load here, then the payment method itself will not load or display anything.  The reason why I commented this out is because it does a console error in the admin.... I need to revisit though, I'm going to add a @todo for a followup.